en_ewt-ud-test.conllu

```diff
@@ -15052,7 +15052,7 @@
 29	.	.	PUNCT	.	_	5	punct	5:punct	_
 
 # sent_id = newsgroup-groups.google.com_n3td3v_e874a1e5eb995654_ENG_20060120_052200-0011
-# text = Please note that neither the e-mail address nor name of the sender have been verified.
+# text = Please note that neither the e-mail address nor name of the sender have been verified.
```

Comment:
I don't know why this line is marked as changed

Reply:
It would appear the original had a non-breaking space here, whereas that is not marked on the CoNLL in any way, so I think it is correct to eliminate it

Reply:
No it's correct to have a non-breaking space: we preserve the original characters in the `text` line as much as possible #418

Reply:
It's not represented on the tokens, though.  How should we do that?

Reply:
Anyway, I can update the #text field easily enough (just pushed that to dev as well)

Reply:
#418 suggests `SpacesAfter`

Reply:
How about like this?

https://github.com/UniversalDependencies/UD_English-EWT/pull/463